modules/ROOT/pages/intro-error-handlers.adoc

```diff
@@ -84,7 +84,9 @@ For the most part, Mule 3 only allows error handling at the flow level, forcing
 
 image::error-handling-try.png[Try Scope]
 
-The error handler behaves as we have explained earlier. In the example above, any database connection errors are propagated, causing the `try` to fail and the flow’s error handler to execute. In this case, any other errors are handled, and the Try scope is considered successful which, in turn, means that the next processor in the flow, an HTTP request, continues its execution.
+In the above example, an on error propagate handler is configured to propagate database connection errors, causing the `Try` to fail and the flow's error handler to execute. 
+An on error continue handler is configured to deal with any other error. The Try scope’s execution is considered successful for that case, which means that the flow’s execution continues with the next processor, an HTTP request.
```

Comment:
something like this to capture the official name and spelling of the components:

```suggestion
On Error Continue component is configured to handle any other error. The Try scope’s execution is successful for that case, which means that the flow’s execution continues with the next processor, an HTTP Request operation.
```

See the On Error component docs for the exact spelling. And for the HTTP Request operation, see the connector docs for the HTTP connector docs.

Reply:
https://docs.mulesoft.com/mule-runtime/4.4/on-error-scope-concept 

https://docs.mulesoft.com/http-connector/1.7/http-request-ref

Reply:
Making the changes. Thank you, Duke!